manage.py

```diff
@@ -74,10 +76,16 @@ def sync():
     seed(include_interventions=True)
 
 
-@manager.command
+@app.cli.command()
+@click.option('--include_interventions', '-i', default=False, help='Include (overwrite) intervention data')
+@click.option('--keep_unmentioned', '-k', default=False, help='Keep orgs and interventions not mentioned in persistence file')
 def seed(include_interventions=False, keep_unmentioned=False):
     """Seed database with required data"""
-    add_static_concepts()
+
+    # Request context necessary for generating data from own HTTP APIs
+    with app.test_request_context():
+        add_static_concepts()
+
```

Comment:
As far as I can tell only `add_static_concepts()` makes HTTP requests against the portal (when adding [valuesets](https://github.com/uwcirg/true_nth_usa_portal/tree/develop/portal/views/fhir.py#L10))